flexget/plugins/input/kitsu.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+
+from flexget import plugin
+from flexget.config_schema import one_or_more
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.utils.cached_input import cached
+from requests import RequestException
+
+log = logging.getLogger('kitsu')
+
+
+class KitsuAnime(object):
+    """Creates an entry for each item in your kitsu.io list.
+    Syntax:
+    kitsu:
+      username: <value>
+      lists:
+      	- <current|planned|completed|on_hold|dropped>
+      	- <current|planned|completed|on_hold|dropped>
+      list_only: <airing|finished>
+      latest: <yes|no>
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'username': {'type': 'string'},
+            'lists': one_or_more({'type': 'string', 'enum': ['current', 'planned', 'completed', 'on_hold', 'dropped']}),
+            'latest': {'type': 'boolean', 'default': False},
+            'limit_to': {'type': 'string', 'enum': ['airing', 'finished']}
+        },
+        'required': ['username'],
+        'additionalProperties': False,
+    }
+
+    @cached('kitsu', persist='2 hours')
+    def on_task_input(self, task, config):
+        entries = []
+        user_payload = {'filter[name]': config['username']}
+        try:
+            user_response = task.requests.get('https://kitsu.io/api/edge/users', params=user_payload)
+            user_response.raise_for_status()
+        except RequestException as e:
+            error_message = 'Error finding User url: {url} status: {status}'.format(
+                url=e.request.url, status=e.response.status_code)
```

Comment:
This is a but note that e doesn't necessarily mean that response attribute. Best to check with `hasattr`